src/textdescriptives/components/quality.py

```diff
@@ -600,6 +636,11 @@ def create_quality_component(
             be considered a top n-gram. Defaults to 3.
         duplicate_n_gram_fraction_range (Tuple[int]): range of n-grams to
             calculate the proportion of duplicate n-grams. Defaults to [5, 10].
+        vocab (Optional[Mapping]): vocabulary to use for calculating the
+            out-of-vocabulary rate. If none, will use the vocabulary of the
+            spaCy model. Note, that small spaCy models do not have a vocabulary.
+            The attribute will only be set if the vocabulary is not None or
+            the spaCy model is medium or large.
```

Comment:
hmm but it does have a vocabulary? It is just dynamic? or? 

We should create a test with a small model